curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b145976723b56a97b8dda.md

```diff
@@ -0,0 +1,54 @@
+---
+id: 657b145976723b56a97b8dda
+title: Task 17
+challengeType: 19
+dashedName: task-17
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Candidate: I have a bachelor's degree in computer science from fCC University. During my studies, I was introduced to various programming languages and software development principles.
+-->
+
+# --description--
+
+The phrase `I was introduced to` uses the simple past tense form `was` of the verb `be`, combined with the past participle `introduced`. This structure, `be` followed by a verb suffixed with `-ed`, is often used to describe actions that happened to the subject in the past. In this case, `I was introduced to` means the candidate was made familiar with something new (programming languages and principles) during their studies.
```

Comment:
```suggestion
The phrase `I was introduced to` uses the simple past tense form `was` of the verb `be`, combined with the past participle `introduced`. This structure, `be` followed by a verb suffixed with `-ed`, is often used to describe actions that happened to the subject in the past. For example:

`At college, I was introduced to many programing languages`
```

Removed the last part to avoid giving the answer.